x-pack/metricbeat/module/aws/_meta/docs.asciidoc

```diff
@@ -0,0 +1,36 @@
+This module periodically fetches monitoring metrics from AWS Cloudwatch.
+
+The default metricset is `ec2`.
+
+[float]
+=== Module-specific configuration notes
+
+This module environment variable `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `AWS_SESSION_TOKEN` and `AWS_REGION`
+references in the config file to set values that need to be configurable during deployment.
+
+There are two different kinds of AWS credentials can be used here: `access keys` and `temporary security credentials`.
+`aws_access_key_id` and `aws_secret_access_key` are the two parts of `access keys` for AWS to authenticate AWS API requests.
+`access keys` are long-term credentials for an IAM user or the AWS account root user. A more AWS recommended way is to use
+`temporary security credentials` instead of `access keys`. `temporary security credentials` consist of an access key ID,
+a secret access key, and a security token which typically returned from `GetSessionToken`. MFA-enabled IAM users would
+need to submit an MFA code while calling `GetSessionToken`. `aws_default_region` is to set the region for SDK to use.
```

Comment:
@ruflin If the user is using `temporary security credentials` then yes, once the token expired, a new token needs to be inserted. Otherwise aws module will stop reporting metrics. Is this approach ok?

Reply:
I don't think we really have an alternative here. What I worry is that the user has to restart Metricbeat in this case except he uses reloading of the module.

What is our recommended lifetime for the session key?

Do we log a good error when the key expires?